geotrek/common/parsers.py

```diff
@@ -113,6 +115,8 @@ def __init__(self, progress_cb=None, user=None, encoding='utf8'):
                 f.name: force_str(f.verbose_name)
                 for f in self.model._meta.many_to_many
             }
+        default_language = self.default_language if self.default_language else settings.MODELTRANSLATION_DEFAULT_LANGUAGE
+        translation.activate(default_language)
```

Comment:
Use `translation.activate` to make sure `name` field is properly linked to the right language `name_fr` field (otherwise we could happen to impact the wrong translation field when using `object.name = "Nom fr"` or `setattr(object, 'name', "Nom fr")`